src/utils/clack-utils.ts

```diff
@@ -697,6 +684,32 @@ export async function runPrettierIfInstalled(): Promise<void> {
       })
       .join(' ');
 
+    if (!changedOrUntrackedFiles.length) {
+      // Likewise, if we can't find changed or untracked files, there's no point in running Prettier.
+      return;
+    }
+
+    const packageJson = await getPackageDotJson();
+    const prettierInstalled = hasPackageInstalled('prettier', packageJson);
+
+    Sentry.setTag('prettier-installed', prettierInstalled);
+
+    if (!prettierInstalled) {
+      return;
+    }
+
+    // prompt the user if they want to run prettier
+    const shouldRunPrettier = await abortIfCancelled(
+      clack.confirm({
+        message:
+          'Looks like you have Prettier in your project. Do you want to run it on your files?',
+      }),
+    );
+
+    if (!shouldRunPrettier) {
+      return;
+    }
+
```

Comment:
No behavior changes here, just made a small refactoring to avoid the double-nested if block